client/src/pages/DetailsPage/item-details.module.scss

```
@@ -2,26 +2,179 @@
 
 
 .itemDetails {
-    .testi1 {
-        background-color: blue;
+    width: 90vw;
```

Das killt die Responsivität (Ich weiß ist jetzt nicht nötig).

Es passt sich zwar der Bildschirm größe an, ist aber nicht ideal falls du einen container hast in dem diese 90vw eben nicht rein passen.

Mit % hingegen belaufen diese 90% immer auf die Größe des Containers.
```suggestion
    width: 90%;
```

---

hast recht. danke! habs korrigiert!